webpack/webpack.config.base.js

```diff
@@ -161,6 +163,12 @@ export const config = {
         loader: 'esbuild-loader',
         options: { tsconfig: './tsconfig.json', target: 'es2020' },
       },
+      {
```

Comment:
I had to add this to allow the use of JSX in some of the library code we are importing e.g. in icons. 

Without this they fail to parse with:
```
ERROR in ./node_modules/.pnpm/@leather-wallet+ui@1.4.4_@babel+core@7.24.4_@babel+preset-env@7.24.6_@babel+core@7.24.4__@swc_jk2kaw66sbdapqloa5m2jvn3oq/node_modules/@leather-wallet/ui/dist-all/src/icons/zap-icon.js 4:16
Module parse failed: Unexpected token (4:16)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```

Reply:
This is actually failing in the CI now with Chromatic so I need to check that